lang/java/reef-wake/wake/src/main/java/org/apache/reef/wake/remote/transport/netty/NettyMessagingTransport.java

```diff
@@ -211,16 +213,22 @@
    * Closes all channels and releases all resources.
    */
   @Override
-  public void close() throws Exception {
+  public void close() {
 
     LOG.log(Level.FINE, "Closing netty transport socket address: {0}", this.localAddress);
 
     this.clientChannelGroup.close().awaitUninterruptibly();
     this.serverChannelGroup.close().awaitUninterruptibly();
-    this.acceptor.close().sync();
-    this.clientWorkerGroup.shutdownGracefully();
-    this.serverBossGroup.shutdownGracefully();
-    this.serverWorkerGroup.shutdownGracefully();
+
+    try {
+      this.acceptor.close().sync();
+    } catch (final Exception ex) {
+      LOG.log(Level.WARNING, "Error closing the acceptor channel for " + this.localAddress, ex);
```

Comment:
Doesn't this mean we leak resources? In that case, we might in fact throw an exception, right?